app/models/recording.rb

```diff
@@ -9,4 +9,10 @@ class Recording < ApplicationRecord
   validates :visibility, presence: true
   validates :length, presence: true
   validates :users, presence: true
+
+  def self.search(input)
+    return where('lower(recordings.name) LIKE ?', "%#{input.downcase}%").includes(:formats) if input
```

Comment:
There's a postgres operator that can be used here instead called `ILIKE` https://kb.objectrocket.com/postgresql/how-to-use-ilike-in-postgres-1258